app/keepers/keepers.go

```diff
@@ -505,6 +511,18 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 		appKeepers.LockupKeeper,
 	)
 
+	// initialize the auction keeper
+	auctionKeeper := auctionkeeper.NewKeeper(
+		appCodec,
+		appKeepers.keys[auctiontypes.StoreKey],
+		appKeepers.AccountKeeper,
+		appKeepers.BankKeeper,
+		appKeepers.DistrKeeper,
+		appKeepers.StakingKeeper,
+		authtypes.NewModuleAddress(govtypes.ModuleName).String(),
```

Comment:
Rewards go to the gov module address :eyes:

Reply:
This is incorrect, it uses a function to send rewards to validators

https://github.com/skip-mev/block-sdk/blob/main/x/auction/rewards/proposer_provider.go#L29-L41